EPRN.Common/Enums/UserRole.cs

```diff
@@ -8,6 +8,7 @@ public enum UserRole
     {
         None,
         Exporter,
-        Reprocessor
+        Reprocessor,
+        ExporterAndReprocessor
```

Comment:
The adding of the value of ExporterAndReprocessor is not needed. The UserRoleService contains a list of roles: if they have both Exporter and Reprocessor, then that are both